projects/pcre2/build.sh

```diff
@@ -0,0 +1,61 @@
+#Build target for libfuzzer
+
+echo '[x] Build target for libfuzzer'
+
+export CC="clang"
+export CFLAGS="-g -fsanitize=fuzzer-no-link,address,bounds,integer,undefined,null,float-divide-by-zero"
+TARGET_FLAGS="-g -fsanitize=fuzzer,address,bounds,integer,undefined,null,float-divide-by-zero"
```

Comment:
`export`?
Also comment on first line can be removed or similar comments should be added for other steps

Reply:
oh i have forgotten.thx